<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendListItem</class>
 <widget class="QWidget" name="FriendListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="profileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>3</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>3</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 255, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>3</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 255, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 255, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>profileLabel</zorder>
  <zorder>nameLabel</zorder>
  <zorder>statusLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
